Expand `Ranked_score` (#8488)
ppy/osu-wiki: wiki/Gameplay/Score/Ranked_score/en.md

# Ranked score

*For other uses, see [Score (disambiguation)](/wiki/Disambiguation/Score).*

The **ranked score** is the total sum of the best scores for all ranked, loved, and approved [difficulties](/wiki/Beatmap/Difficulty) played online.

> - History  (6 October 2007 – 24 July 2012)

Reviewer: would rather just have a clean heading like this. the timeline is conveyed by the below paragraph well enough i think

```suggestion
## History
```

> Prior to [ppv1](/wiki/Performance_points/ppv1) and [ppv2](/wiki/Performance_points/ppv2), a player's rank was determined by the amount of ranked score the player had. thus resulting in frequent #1 changes due to how small osu! was at the time and the lack of maps. Ranked score was fully replaced by ppv1 as of July 24, 2012, which was later to be replaced by ppv2 on January 27, 2014.

Reviewer: ```suggestion
Prior to [ppv1](/wiki/Performance_points/ppv1) and [ppv2](/wiki/Performance_points/ppv2), a player's [rank](/wiki/Ranking) was determined by the amount of ranked score the player had, thus resulting in frequent #1 changes due to how small the osu! community was at the time and the lack of maps. Ranked score was fully replaced by ppv1 as of July 24, 2012, which was later to be replaced by ppv2 on January 27, 2014.
```

> ## Approved beatmaps
> 
> Approved was a category for beatmaps that awarded a huge amount of score or maps that were too difficult to be ranked (e.g.,
> [marathons](/wiki/Beatmap/Marathon) Approved beatmaps formerly did not award ranked score, but instead awarded [total score](/wiki/Gameplay/Score/Total_score) and pp. In contrast, "loved maps" do not award pp but do award both ranked and total score.

Reviewer: we already have https://osu.ppy.sh/wiki/en/Beatmap/Category#approved so i suggest condensing this to only the relevant bits

```suggestion
Beatmaps that exceeded length and maximum score requirements on older renditions of the [ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria) were able to instead be moved into the [Approved category](https://osu.ppy.sh/wiki/en/Beatmap/Category#approved). Such beatmaps did not award ranked score at the time, but instead awarded [total score](/wiki/Gameplay/Score/Total_score) and [performance points](/wiki/Performance_points/).
```

Author: Fair

Author: `[Approved category](https://osu.ppy.sh/wiki/en/Beatmap/Category#approved)`
why is this a full URL?
